eclass/autotools.eclass

```diff
@@ -31,7 +31,7 @@ case ${EAPI} in
 		# Needed for eqawarn
 		inherit eutils
 		;;
-	7) ;;
+	7|8) ;;
```

Comment:
this needs to be put on hold, there's some other issues that need to be resolved first